api/client/group.go

```diff
@@ -104,6 +104,17 @@ func (g *orderedListGroup) AuthSign(req, id []byte, provider auth.Provider) (res
 	return nil, err
 }
 
+func (g *orderedListGroup) BundleAuthSign(req, id []byte, provider auth.Provider) (resp []byte, err error) {
```

Comment:
Just a personal coding preference: I would've created a private authSign method which would actually use AuthSign or BundleAuthSign depending on a boolean, so that code wouldn't be repeated verbvatim. But I get you wanted to reduce the patch surface as much as possible, so really your choice!

Reply:
Yeah, I thought it might be wise to go with minimal surface.